src/pages/Main/Main.js

```diff
@@ -1,7 +1,124 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import MainCarousel from "./MainCarousel/MainCarousel";
+import styled from "styled-components";
+import ClassCarousel from "./ClassCarousel/ClassCarousel";
+import MainProducts from "./MainProducts/MainProducts";
+import { useNavigate } from "react-router-dom";
+import { config } from "../../config";
 
 function Main() {
-  return <div>Main</div>;
+  const [products, setProducts] = useState([]);
+  const [categoryProducts, setCategoryProducts] = useState([]);
+  const [userLearn, setUserLearn] = useState([]);
+
+  const navigate = useNavigate();
+
+  function goToDetail(id) {
+    navigate(`/productDetail/${id}`);
+  }
+
+  useEffect(() => {
+    fetch(`/data/MainMock/mainCategory.json`)
+      // fetch(`${config.main}`)
+      .then(res => res.json())
+      .then(data => setProducts(data.result));
+  }, []);
+
+  useEffect(() => {
+    fetch(`${config.list}`)
```

Comment:
```suggestion
    fetch(config.list)
```
* import 해 왔기 때문에 template literal 없이도 가능 합니다

Reply:
오오,, 이게 문자열 그대로도 인식을 하네요.
리터럴을 써야 변수처럼 쓸 수 있을 것 같아서 지정해둔건데 감사합니다,,!